maze-monitors.patch

```diff
@@ -0,0 +1,33 @@
+diff --git a/game.py b/game.py
+index 7bfa351..471da01 100644
+--- a/game.py
++++ b/game.py
+@@ -43,6 +43,8 @@ from maze import Maze, Rectangle
+ from player import Player
+ import sensors
+ 
++from sugarapp.helpers import PrimaryMonitor
++
+ 
+ class MazeGame(Gtk.DrawingArea):
+     """Maze game controller.
+@@ -79,7 +81,7 @@ class MazeGame(Gtk.DrawingArea):
+         # keep a list of all players, local and remote,
+         self.allplayers = [] + self.localplayers
+ 
+-        screen = Gdk.Screen.get_default()
++        screen = PrimaryMonitor
+         self.aspectRatio = float(screen.width()) / screen.height()
```

Comment:
Just use `float(PrimaryMonitor.width()) / PrimaryMonitor.height() ` here.

Reply:
resolved in 6aff0b44f72007ea537a6e74ed4f79bc8e450481